setup.py

```diff
@@ -31,32 +31,19 @@ def package_files(directory):
     packages=find_packages(exclude=["tests", "tests.*"]),
     package_data={"mlflow": js_files + models_container_server_files + alembic_files},
     install_requires=[
-        "alembic<=1.4.1",
-        # Required
-        "azure-storage-blob",
-        "click>=7.0",
-        "cloudpickle",
-        "databricks-cli>=0.8.7",
-        "requests>=2.17.3",
-        "six>=1.10.0",
-        'waitress; platform_system == "Windows"',
-        'gunicorn; platform_system != "Windows"',
-        "Flask",
-        "numpy",
-        "pandas",
```

Comment:
I noticed a lot of attention devoted to `numpy`, `pandas`, and `sqlparse` throughout other portions of this skinny client refactoring. Is it absolutely necessary to exclude these libraries? They aren't currently versioned and are fundamental to most data science / ML workflows anyway. Have we had any higher-level conversations or done any design regarding what should and should not be included in a skinny client?

Reply:
I do not think it is absolutely necessary.  sqlparse, if it is a small dependency could be included back to enable search_runs support. I think there was a general push to have the DS specific libraries removed, however, as you noticed from the PR, even lazy loading those packages involves a lot of code changes.

Reply:
I do not think it is absolutely necessary. sqlparse, if it is a small dependency could be included back to enable search_runs support. I think there was a general push to have the DS specific libraries removed, however, as you noticed from the PR, even lazy loading those packages involves a lot of code changes.

Reply:
Got it - can we put together a list of requirements for the skinny client that will help us make decisions on which libraries are / are not necessary? (I.e. it'd be awesome to understand what the general push looks like). I think @akshaya-a had done some work here? Would be excited to meet and discuss this as well!